src/components/NcListItem/NcListItem.vue

```diff
@@ -641,7 +640,7 @@ export default {
 		 */
 		actionsAriaLabel: {
 			type: String,
-			default: '',
+			required: true,
```

Comment:
I would consider this more or less a breaking change :eyes:  As this is more strict.
Maybe just add a note that this will be required in v9 and instead of making it required add:
```suggestion
			default: t('Actions'),
```

Reply:
True, I would be fine if we merge that in v9 only. I don't think a default aria label would be a good thing, this would mess up accessibility.

Is v9 the `next` branch?

Reply:
leaving it unset will trigger the default `NcActions` label which is also just `Actions` so this would also be fine

Reply:
> Is v9 the `next` branch?

yes

Reply:
Alright, changed branch to target next.

Reply:
@artonge As @susnux mentioned, we don't need to require aria label here. `NcActions` has the default label already, and it works fine by default. So there is no need to make users always create a name for the menu when using `NcListItem` and make it a breaking change.

Reply:
But having a default "Actions" label for accessibility is not really accessible, no?

Reply:
> But having a default "Actions" label for accessibility is not really accessible, no?

Why not?

Reply:
> have x elements with the same description on the same page

It is not a problem, they have different contexts. It is not an "Actions" button somewhere on the page. It is a list item, and withing the list item there is a menu button with actions called "Actions". In a general case it's enough.

When a person uses only a screen-reader with no screen, they go to this button throw the page and list items. If a person uses a screen together with assistive technologies, then they know with which part of the UI they interact.

Like if we have a normal button to delete a list item in the list item with a normal text content, we don't write text "Delete user X" or "Delete list item #125". We just call it a "Delete" button.

Reply:
Best rule I got told: Both should have the same information (visual vs accessible).
So we have visual information: Actions button.
So accessible information "Actions" would be ok. The context is already provided by the wrapping element.